admin/environment-management/workspace-providers.md

```diff
@@ -4,17 +4,35 @@ description: Learn how workspace providers can minimize latency for users.
 state: beta
 ---
 
-Workspace Providers are a forthcoming feature that enables a single Coder
-deployment to provision and manage workspaces across multiple Kubernetes
-clusters and namespaces, including ones located in other geographies, regions,
-or clouds.
+Workspace Providers are logical groups of resources developers can deploy
+workspaces onto. Workspace Providers enable a single Coder deployment to
+provision and manage workspaces across multiple Kubernetes clusters and
+namespaces, including ones located in other geographies, regions, or clouds.
 
 Distributed teams can use this feature to allow users to manage workspaces in
 the cluster that's nearest, reducing network latency and improving developers'
 experience. Additionally, you can use workspace providers to support data
 sovereignty requirements or increase the isolation between workspaces running in
 the same region or cluster.
 
+### Built-In Workspace Provider
+
+By default, all Coder deployments will have a `built-in` Workspace Provider that
+identifies the kubernetes cluster the Coder deployment is in. This enables
+workspaces to be deployed into the same cluster as the Coder deployment with no
+additional actions required.
+
+### Remote Workspace Providers
+
+A Workspace Provider can be deployed into any existing kubernetes cluster, and
```

Comment:
Question: does "Workspace Provider" refer to the thing providing the infrastructure (e.g. a Kubernetes cluster) or to the agent connected to the thing providing the infrastructure? This reads like the latter, but from a users' perspective, Kubernetes or EC2 is providing the workspaces, right?

Perhaps:

> You can deploy a Workspace Provider in any existing Kubernetes cluster, and developers will be able to create workspaces in that infrastructure, provided that they have permissions to do so.

Reply:
Sounds like this docs could use an architecture diagram. In my mind, "Workspace Provider" is the functional unit, made up of a envproxy and a cluster we deploy into. I just am hesitant to add in terminology for the envproxy itself, so I wanted to keep it abstract and say "deploy the thing and things work"